### DISCUSSION TITLE
Best practice for accessing Values/OneOrMany objects?

### DISCUSSION THREAD

**Discussion by kirk-marple** [RehanSaeed/Schema.NET#44] [Q&A]
Where 'album' is MusicAlbum, and Author is set as a Person.

Is there a better way to access the author name than this?

album.Author.Value.Value2.FirstOrDefault().Name.Value.FirstOrDefault()

Thanks!

**Comment by RehanSaeed** [comment]
I've just checked in some code that will allow you do do this:

```
List<Person> people = album.Author;
```

This is using implicit conversion operators. Thanks for the idea.
